### PR TITLE
feat: support list users

### PR DESCRIPTION
**Pull request by evansims**
## Description

- Parent Epic: https://github.com/openfga/sdk-generator/issues/329

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [x] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ This is an autogenerated python SDK for OpenFGA. It provides a wrapper around th
       - [Expand](#expand)
       - [List Objects](#list-objects)
       - [List Relations](#list-relations)
+      - [List Users](#list-users)
```
Comment (ewanharris) [inline README.md:41]: I think we're maybe missing the docs content in the readme for this?

**Review by ewanharris — COMMENTED** [review]
Nice @evansims! I've just the comment around the readme content but otherwise it looks good to me.

File: README.md (hunk 1 above)
Comment (evansims) [reply README.md:41]: Good catch, thanks! Fixed!

**Review by ewanharris — APPROVED** [review]
Looks good to me! 🎉

**Pull request merged**